### ISSUE TITLE
How do I identify the paper's novelty in the code?

### ISSUE THREAD

**Issue by ConceptualCode** [mk-minchul/AdaFace#78]
How do I identify the paper's novelty in the code?

I want to use the novelty to train in a different architecture. Please how do I do that?

You mentioned the Image Quality Indicator and Adaptive Margin Function in your proposed design. Where did you write them in the code?
![adaface design](https://user-images.githubusercontent.com/52907793/205663024-7dcd77f1-e2bd-49c1-87ad-243be65225f3.png)

**Comment by mk-minchul** [comment]
The AdaFace loss function is defined in [head.py#L45](https://github.com/mk-minchul/AdaFace/blob/9507e0abeb4092c06798252f9efc059511cc9e64/head.py#L45).
The image quality indicator is the feature norm which is calculated in [net.py](https://github.com/mk-minchul/AdaFace/blob/9507e0abeb4092c06798252f9efc059511cc9e64/net.py#L329)
